compiler/rustc_hir_analysis/src/lib.rs

```diff
@@ -529,8 +529,6 @@ pub fn check_crate(tcx: TyCtxt<'_>) -> Result<(), ErrorGuaranteed> {
         tcx.hir().for_each_module(|module| tcx.ensure().check_mod_item_types(module))
     });
 
-    tcx.sess.time("item_bodies_checking", || tcx.typeck_item_bodies(()));
```

Comment:
What is going to ensure that typecheck errors for all functions are reported without this call?

Reply:
This also regresses the quality of the output of `-Ztime-passes`.

Reply:
as mentioned in the main post

> This phased order of query invocations was already a lie, as any query that looks at types (e.g. the wf checks run before) can cause e.g. const eval which invokes borrowck, which invokes typeck, ...

we already didn't produce correct time tracking for it